documentation/modules/con-console.adoc

```diff
@@ -7,6 +7,6 @@
 
 {ConsoleName} uses the OpenShift RBAC permissions model. For more information about the OpenShift RBAC permissions model, see the link:https://docs.openshift.com/container-platform/3.11/admin_guide/manage_rbac.html[OpenShift documentation^].
 
-To access the {ConsoleName}, the OpenShift user requires `addressspace` and `address` roles. The specific permissions given to the role govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the role associated with the user, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
+To access the {ConsoleName}, the OpenShift user requires the `addressspace` and `address` resources to be included in the role object, which defines a set of permissions that apply to one or more resources. The specific permissions given to the role object govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the associated role object, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
```

Comment:
Mmm.  I'm not sure about the first sentence.  It is not wrong but overly detailed.  We are already supplying a link to 3.11/admin_guide/manage_rbac.html so the user can easily get the detail from the authoritative source.

How about:

"To use the {ConsoleName}, the OpenShift user requires a role that grants access to `addressspace` and `address` resources."